src/main/java/fi/hsl/transitdata/pubtransredisconnect/QueryUtil.java

```diff
@@ -0,0 +1,142 @@
+package fi.hsl.transitdata.pubtransredisconnect;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+
+public class QueryUtil {
+
+    private static final Logger log = LoggerFactory.getLogger(QueryUtil.class);
+
+    public static final String DVJ_ID = "dvj_id";
+    public static final String DIRECTION = "direction";
+    public static final String ROUTE_NAME = "route";
+    public static final String START_TIME = "start_time";
+    public static final String OPERATING_DAY = "operating_day";
+    public static final String STOP_NUMBER = "stop_number";
+
+    private static QueryUtil ourInstance = new QueryUtil();
+
+    public static QueryUtil getInstance() {
```

Comment:
Providing an instance of a class which only has static methods and no shared data seems a bit odd. I don't see a need for singleton pattern here, I guess these could just be removed..?

Reply:
Good catch, auto-generated.